src/main/java/frc/lib/swerve/Swerve.java

```diff
@@ -52,4 +54,8 @@ public Command zeroGyroCommand() {
   public double getPitch() {
     return m_pigeon2.getPitch().getValueAsDouble();
   }
+  
+  public void targetAngleDrive(Rotation2d moduleDirection){
+    setControl((new SwerveRequest.PointWheelsAt()).withModuleDirection(moduleDirection));
```

Comment:
This is not the correct control. This will point al the modules to the target angle, not the robot